blueprint/core/blueprint_ReactApplicationRoot.cpp

```diff
@@ -135,30 +120,25 @@ namespace blueprint
     //==============================================================================
     void ReactApplicationRoot::handleRuntimeError(const EcmascriptEngine::Error& err)
     {
-#if ! JUCE_DEBUG
+       #if ! JUCE_DEBUG
         // In release builds, we don't catch errors and show the red screen,
         // we allow the exception to raise up to the user to be handled properly
         // for a production app.
         throw err;
-#endif
+       #endif
 
         JUCE_ASSERT_MESSAGE_THREAD
 
-        DBG("");
-        DBG("==== Error in JavaScript runtime. Context: ====");
-        DBG(err.context);
-        DBG("");
-        DBG(err.what());
+        juce::Logger::writeToLog ("\n==== Error in JavaScript runtime. Context: ====\n" + err.context + juce::newLine + err.what());
```

Comment:
Using `DBG` isn't great from a user's point of view, and in fact it might not even be helpful in a release build. So, I swapped these bits out for a log.